middleware/auth.js

```diff
@@ -1,25 +1,22 @@
-const jwt = require("jsonwebtoken");
 const userService = require("../services/userService");
+const jwt = require("jsonwebtoken");
```

Comment:
```suggestion
const jwt = require("jsonwebtoken");

const userService = require("../services/userService");
```
외부 모듈인 jwt 먼저 작성될 수 있게 순서 바꿔주세요~